client/src/App.vue

```diff
@@ -18,7 +18,7 @@ export default {
     Header
   },
   beforeUpdate() {
-    this.$store.dispatch('fetchUser');
+    // this.$store.dispatch('fetchUser');
```

Comment:
Can you throw a TODO in on this so we remember why we commented it out?

Reply:
Removed commented out code